src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -480,9 +573,22 @@ private partial class KeyValueListNode
             public KeyValueListNode Next;
         }
 
+        private static ActivityIDFormat s_DefaultIdFormat;
+
         private KeyValueListNode _tags;
         private KeyValueListNode _baggage;
+        private string _traceState;
         private bool isFinished;
         #endregion // private
     }
+
+    /// <summary>
+    /// The possibilities for the format of the ID
+    /// </summary>
+    public enum ActivityIDFormat : byte
+    {
+        Unknown,     // ID format is not known.     
+        RequestID,   // |XXXX.XX.XX ...
```

Comment:
should `RequestID` be called `HierarchicalRequestID`? `|` tells that it is hierarchical, anything else - it's flat.

Reply:
Also place use this example in comment so it will be clear that underscores are expected: `|XXXX.XX_YY.XX_YY ...`